src/audio/OpenALAudioEngine.cpp

```diff
@@ -0,0 +1,488 @@
+#ifdef RE_USE_SOUND
+
+#include "OpenALAudioEngine.h"
+
+#include <algorithm>
+#include <array>
+#include <atomic>
+#include <mutex>
+#include <thread>
+
+#include <AL/al.h>
+#include <AL/alc.h>
+
+using namespace Audio;
+
+static std::vector<std::string> enumeratedDevices;
```

Comment:
maybe instead of global `static` use anonymous namespace?

Reply:
I don't really see the need, it's just going to be used in this file only, I think. Actually, it could even be declared as static inside the function that uses it 🤔

Reply:
Even better, we could also stick to the YAGNI principle and remove the functionality outright...!